build.gradle

```diff
@@ -372,9 +385,38 @@ task createApiDocsRedirect {
   }
 }
 
+task publishDocs {
+  dependsOn 'closeAndReleaseRepository'
+  doLast {
```

Comment:
I think the process of cloning a fresh tmp_gh-pages dir is best and least susceptible to local state.